packages/vega-schema/src/autosize.js

```diff
@@ -24,6 +24,10 @@ const autosize = oneOf(
   })
 );
 
+const DESCRIPTION = "Sets how the visualization size should be determined. If a string, should be one of `\"pad\"`, `\"fit\"` or `\"none\"`.\nObject values can additionally specify parameters for content sizing and automatic resizing.\n`\"fit\"` is only supported for single and layered views that don't use `rangeStep`.\n\n__Default value__: `pad`";
+
+Object.assign(autosize, { description: DESCRIPTION });
```

Comment:
I think it would be better to put the text into the object itself rather than assigning it later.

Reply:
@domoritz The text can be too long. That is why I created a new variable.

Reply:
I still think we can avoid the assign.

Reply:
@domoritz okay I will remove the `DESCRIPTION` variable.

Reply:
@domoritz Done